db/migrations/20230207110032_create_products_table.sql

```diff
@@ -0,0 +1,16 @@
+-- migrate:up
+CREATE TABLE products (
+  id int PRIMARY KEY NOT NULL AUTO_INCREMENT,
+  name varchar(255) NOT NULL,
+  description varchar(2700),
+  subcategory_id int NOT NULL,
+  price decimal,
+  discount_price decimal,
+  product_img varchar(2700),
```

Comment:
`image_url`로 수정해주세요.
```suggestion
  image_url varchar(2700),
```